https/tls_config.go

```diff
@@ -168,3 +218,44 @@ func Listen(server *http.Server, tlsConfigPath string, logger log.Logger) error
 	}
 	return server.ListenAndServeTLS("", "")
 }
+
+// For go1.14, unmarshalling is done properly in tls_ciphers.go.
+type cipher uint16
+
+type curve tls.CurveID
+
+func (c *curve) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var s string
+	err := unmarshal((*string)(&s))
+	if err != nil {
+		return err
+	}
+	switch strings.ToUpper(s) {
```

Comment:
We don't munge cases elsewhere, let's not do it here.

Reply:
removed